packages/mutation-testing-elements/package-lock.json

```diff
@@ -24,6 +24,22 @@
       "resolved": "https://registry.npmjs.org/lodash.groupby/-/lodash.groupby-4.6.0.tgz",
       "integrity": "sha1-Cwih3PaDl8OXhVwyOXg4Mt90A9E=",
       "dev": true
+    },
+    "mutation-testing-metrics": {
```

Comment:
I think you ran `npm install` directly in `packages/mutation-testing-elements`. Please run `npm install` (which in turn runs `lerna bootstrap`) in the root of the project to fix this.